### DISCUSSION TITLE
RenderSettings 에러

### DISCUSSION THREAD

**Discussion by yoonjoohwan** [naverz/zepeto-studio-kor#761] [5. Q&A]
Light 를 만지고 있습니다
        RenderSettings.ambientLight = this.ambientColor.Evaluate(timePercent);
        RenderSettings.fogColor = this.FogColor.Evaluate(timePercent);
를 실행시키면 
![image](https://user-images.githubusercontent.com/91707090/168545466-41f88a3a-20c3-49b8-a285-9df8ed10cebe.png)
해당의 에러가 뜹니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 버그 리포트는 이슈 트래킹 및 진행 상황 파악을 위해 이슈 게시판에 업로드 부탁 드립니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/700
감사합니다.
